projects/aca-content/src/lib/components/search/search-results/search-results.component.spec.ts

```diff
@@ -49,6 +54,10 @@ describe('SearchComponent', () => {
   const searchRequest = {} as SearchRequest;
   let params: BehaviorSubject<any>;
   let showErrorSpy: jasmine.Spy;
+  let loader: HarnessLoader;
+
+  const editSavedSearchesSpy = jasmine.createSpy('editSavedSearch');
+  const getSavedSearchButton = () => fixture.nativeElement.querySelector('.aca-content__save-search-action');
```

Comment:
can you set type of return for this function?

Reply:
Added